posthog/hogql/functions/sparkline.py

```diff
@@ -2,11 +2,19 @@
 
 
 def sparkline(node: ast.Expr, args: list[ast.Expr]) -> ast.Expr:
-    return ast.Tuple(
-        exprs=[
-            ast.Constant(value="__hx_tag"),
-            ast.Constant(value="Sparkline"),
-            ast.Constant(value="data"),
-            args[0],
-        ]
-    )
+    exprs: list[ast.Expr] = [
+        ast.Constant(value="__hx_tag"),
+        ast.Constant(value="Sparkline"),
+        ast.Constant(value="data"),
+        args[0],
+    ]
+
+    if len(args) == 2:
```

Comment:
Going to swap this out for the HogQLX component on the frontend so we don't need conditional argument "labels". Is this the best way of doing conditional labels though?